integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDiiSample.java

```diff
@@ -0,0 +1,419 @@
+                                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                            domainName,
+                            domainNamespace,
+                            condition.getElapsedTimeInMS(),
+                            condition.getRemainingTimeInMS()))
+            .until(domainExists(domainName, DOMAIN_VERSION, domainNamespace));
+
+    final String adminServerName = "admin-server";
+    final String adminServerPodName = domainName + "-" + adminServerName;
+
+    final String managedServerNameBase = "managed-server";
+    String managedServerPodNamePrefix = domainName + "-" + managedServerNameBase;
+    int replicaCount = 2;
+
+    // verify the admin server service and pod is created
+    checkPodReadyAndServiceExists(adminServerPodName, domainName, domainNamespace);
+
+    // verify managed server services created and pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      checkPodReadyAndServiceExists(managedServerPodNamePrefix + i, domainName, domainNamespace);
+    }
+    checkAccessToEMconsole(adminServerPodName);
+  }
+
+  // copy samples directory to a temporary location
+  private static void setupSample() {
+    assertDoesNotThrow(() -> {
+      logger.info("Deleting and recreating {0}", tempSamplePath);
+      deleteDirectory(tempSamplePath.toFile());
+      Files.createDirectories(tempSamplePath);
+
+      logger.info("Copying {0} to {1}", samplePath, tempSamplePath);
+      copyDirectory(samplePath.toFile(), tempSamplePath.toFile());
+    });
+
+    String command = "chmod -R 755 " + tempSamplePath;
+    logger.info("The command to be executed: " + command);
+    assertTrue(new Command()
+        .withParams(new CommandParams()
+            .command(command))
+        .execute(), "Failed to chmod tempSamplePath");
+  }
+
+  private void updateDomainInputsFile(String domainUid, Path sampleBase, String script) {
+    // in general the node port range has to be between 30,100 to 32,767
+    // to avoid port conflict because of the delay in using it, the port here
+    // starts with 30172
+    final int t3ChannelPort = getNextFreePort(30172, 32767);
+    final int adminNodePort = getNextFreePort(30701, 32767);
+
+    // change namespace from default to custom, domain name, and t3PublicAddress
+    assertDoesNotThrow(() -> {
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
+          "mode: wdt", "mode: " + script);
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
+          "domainHomeImageBase: container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4",
+          "domainHomeImageBase: " + FMWINFRA_IMAGE_TO_USE_IN_SPEC);
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain.sh").toString(),
+          "imagetool.sh update",
+          "imagetool.sh update\n  --buildNetwork host");
```

Comment:
Why do not you ask Leny to modify the create-domain.sh script to make change to imagetool update command to “imagetool.sh update --buildNetwork host” permanently instead of we modifying the script  create-domain.sh on the fly.  See if there is any side effect by making this permanent.